storage/src/backend/registry.rs

```diff
@@ -260,7 +260,7 @@ impl RegistryState {
 
         let token_resp = connection
             .call::<&[u8]>(
-                Method::GET,
+                Method::POST,
```

Comment:
Could you please provide some references proving POST method is more generic?  They are too different. 🤣

Reply:
Will ref the [codes](https://github.com/containerd/containerd/blob/878132923db35f31c8c7b324d9cd1e5f59046064/remotes/docker/auth/fetch.go#L117), but we still need more [tests](https://github.com/dragonflyoss/image-service/pull/764/files).

Reply:
from https://docs.docker.com/registry/spec/auth/oauth/

> Not all token servers implement oauth2. If the request to the endpoint returns 404 using the HTTP POST method, refer to [Token Documentation](https://docs.docker.com/registry/spec/auth/token/) for using the HTTP GET method supported by all token servers.